### ISSUE TITLE
Use requests rather than PyCurl

### ISSUE THREAD

**Issue by Miserlou** [Eyevinn/hls-download#3]
Is there any reason for using the platform-dependent PyCurl versus the usual `requests`?

**Comment by birme** [comment]
No, not a really good reason. Would be happy to receive a PR for that :)
